pkg/new-ui/v1beta1/hp.go

```diff
@@ -27,14 +27,45 @@ import (
 
 	commonv1beta1 "github.com/kubeflow/katib/pkg/apis/controller/common/v1beta1"
 	api_pb_v1beta1 "github.com/kubeflow/katib/pkg/apis/manager/v1beta1"
+	"github.com/pkg/errors"
```

Comment:
Should we stop using this library since this has been archived?

https://github.com/pkg/errors

Reply:
switched to Go's build-in errors package